modules/wordads/wordads.php

```diff
@@ -496,6 +514,17 @@ function get_ad_snippet( $section_id, $height, $width, $location = '', $css = ''
 		if ( ! empty( self::$ad_location_ids[ $location ] ) ) {
 			$loc_id = self::$ad_location_ids[ $location ];
 		}
+		if ( class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ) {
```

Comment:
We may want to consider extracting this into a separate `is_amp` function to include support for various versions of AMP solutions self-hosted sites might use.

Unless we're firm on only supporting the Jetpack_AMP_Support version, then nevermind :)

Reply:
As noted in https://github.com/Automattic/jetpack/pull/12733#issuecomment-503255093, let's see what Jetpack folks think?